includes/wc-update-functions.php

```diff
@@ -1411,6 +1411,46 @@ function wc_update_320_db_version() {
 }
 
 /**
+ * Move comment based webhook logs to WC_Logger.
+ *
+ * @return void
+ */
+function wc_update_330_webhook_logs() {
```

Comment:
Since the logs are limited to 25 comments and rotate, do we need to migrate this data or just dump it?

@claudiosanches

Reply:
@kloon @mikejolley no need for this, since my migration script will remove all comments after migrated to CRUD.